packages/bull-vault/src/FlashBull.sol

```diff
@@ -181,6 +180,10 @@ contract FlashBull is UniFlash {
         IERC20(bullStrategy).transfer(msg.sender, IERC20(bullStrategy).balanceOf(address(this)));
     }
 
+    /**
+     * @notice flash withdraw from strategy, receiving ETH, providing wSqueeth and USDC, and providing strategy tokens
+     * @param _params FlashWithdrawParams struct
+     */
```

Comment:
think for flashwithdraw event we need to know the eth amount received by sender in end not just the params

Reply:
yep added